board/qualcomm/qcom-sdm660-phone.config

```diff
@@ -0,0 +1,58 @@
+# Phone-friendly environment
+CONFIG_ENV_DEFAULT_ENV_TEXT_FILE="board/qualcomm/qcom-sdm660-phone.env"
+
+# Support booting Android boot images
+CONFIG_BOOTMETH_ANDROID=y
+CONFIG_CMD_ABOOTIMG=y
+
+# "Nice to have"
+CONFIG_CMD_POWEROFF=y
+CONFIG_CMD_UFETCH=y
+CONFIG_CMD_SELECT_FONT=y
+CONFIG_CMD_PMIC=y
+CONFIG_VIDEO_FONT_4X6=y
+CONFIG_VIDEO_FONT_8X16=y
+CONFIG_VIDEO_FONT_SUN12X22=y
+# SDM630/636/660 don't have UFS, they need eMMC/SD card support
+CONFIG_SD_BOOT=y
+CONFIG_QCOM_UFS=n
+CONFIG_CLK_QCOM_SDM660=y
```

Comment:
maybe add a comment here like
`# non-default SOC support options`

Reply:
Rebased on fresh master, dropped `CONFIG_PINCTRL_QCOM_SDM660` and added a TODO about `CONFIG_CLK_QCOM_SDM660` that it should be in `qcom_defconfig`